### ISSUE TITLE
Transition Duration of 0 not working

### ISSUE THREAD

**Issue by ariel-frischer** [YIZHUANG/react-multi-carousel#166]
**Describe the bug**
Unable to turn off transitions, I've tried changing transitionDuration to 0 or very small numbers, 1, 0.1, etc.. the transition duration does not seem to change it takes about a half second to complete the transition. I've also tried giving customTransitions: `all 1ms`, `none`, `transform 0ms ease-in-out` nothing seems to do the trick...
**To Reproduce**
Try setting transitionDuration to 0.

**Expected behavior**
No transition when transition duration is set to 0.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Chrome
 - Version 2.5.5

**Comment by michaelyuen** [comment]
### TLDR: Are you sure `customTransition="none"` doesn't work for you?

---

### Investigation
I think there is a bug. Reference [this line](https://github.com/YIZHUANG/react-multi-carousel/blob/master/src/Carousel.tsx#L89):
`props.transitionDuration || defaultTransitionDuration`

If `transitionDuration` is 0, it would evaluate to false and `defaultTransitionDuration` will be used.

---

### What if you try `transitionDuration={1}`? (so it will evaluate to true)
- (I'm using the carousel in a lightbox) When the lightbox is opened, there is no observable transition. ✅ 
- After opening, navigating to the previous or next slide, there is a transition (with duration) ❌ 

#### Why?
_I think_, the code [found here](https://github.com/YIZHUANG/react-multi-carousel/blob/master/src/Carousel.tsx#L727-L731):
```
transition: this.isAnimationAllowed ? customTransition || defaultTransition : "none",
```
Glancing through the source, `isAnimationAllowed` looks to be internal and for the intent of this discussion consider the value to be `true`. Thus, it will look to `customTransition` first and fallback to `defaultTransition` , and not consider `transitionDuration` at all. 🤷‍♂️ 

If `customTransition` isn't provided, it'll end up as `transform 400ms ease-in-out` - the `defaultTransition`.

---

### What if you try `customTransition="none"`?
- For me, this does the trick. The carousel has no transition. ✅ 

---

After looking around the source while writing this up, I suspect that you and I were both interpreting the function of `transitionDuration` incorrectly. I don't think it's intended to be used independently. I think it's actually a prop that is required **in conjunction** with `customDuration`, to sync timing behavior with the custom transition, resizing, etc. Perhaps someone else more knowledgeable will chime in.

**Comment by ArielFrischer** [comment]
Ok nevermind, I think it works there was some class css that was overriding the transitions which I did not notice. My mistake thank you for the help!

**Comment by michaelyuen** [comment]
I don't know if you can (I can't), but if this is resolved please close the issue. 🙏
